packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -114,11 +134,36 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
                 )}
             </BigNumberHalf>
 
-            <BigNumberHalf>
-                <BigNumberLabel $fontSize={labelFontSize}>
-                    {bigNumberLabel || defaultLabel}
-                </BigNumberLabel>
-            </BigNumberHalf>
+            {showLabel ? (
+                <BigNumberHalf>
+                    <BigNumberLabel $fontSize={labelFontSize}>
+                        {bigNumberLabel || defaultLabel}
+                    </BigNumberLabel>
+                </BigNumberHalf>
+            ) : null}
+
+            {showComparison ? (
+                <BigNumberHalf>
+                    <BigNumber
+                        $fontSize={comparisonFontSize}
+                        style={{
+                            marginTop: 10,
+                        }}
+                    >
+                        {typeof comparisonValue === 'number' &&
+                        comparisonFormat === formatValues.PERCENTAGE
+                            ? `${comparisonValue}%`
+                            : comparisonValue}
+                        {typeof comparisonValue === 'number' &&
+                        comparisonValue > 0 ? (
+                            <IconArrowUpRight style={{ marginLeft: 5 }} />
+                        ) : typeof comparisonValue === 'number' &&
+                          comparisonValue < 0 ? (
+                            <IconArrowDownRight style={{ marginLeft: 5 }} />
+                        ) : null}
```

Comment:
Based on the requested changes, we shouldn't rely on the `comparisonValue` type since we are going to format the value there to a string. It would also simplify this code since the business logic is all in the hook.

I propose something like this:
The hook only returns the properties:
```
comparisonValue: string; // eg: '$200'
comparisonDiffType: 'positive' | 'negative' | 'none'; // we can create a new enum for this
```
We render
```
<BigNumber ... >
   {comparisonValue}
   { comparisonDiff === 'positive' && <IconArrowUpRight ... /> }
   { comparisonDiff === 'negative' && <IconArrowDownRight ... /> }
</BigNumber>
```

Reply:
sleek, I was thinking of a refactor along those lines! thx 👍🏼